composerun.py

```diff
@@ -0,0 +1,57 @@
+import argparse
```

Comment:
Maybe just remove this file? Nice for dev.. but a bit overkill instead of just having a local .env file or similar?